assets/prototype/domain/shared/predicates/shared/selectionPredicates/index.test.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'ramda';
+
+/**
+ * Internal dependencies
+ */
+import { entityDbId, entityGuId, entityHasDbId } from './index';
+import { nodes as datetimes } from '../../../../../domain/eventEditor/data/queries/datetimes/test/data';
+
+describe('entityDbId', () => {
+	it('should return dbId for each entity', () => {
+		datetimes.forEach((entity) => {
+			expect(entityDbId(entity)).toBe(entity.dbId);
+		});
+	});
+});
+
+describe('entityGuId', () => {
+	it('should return id for each entity', () => {
+		datetimes.forEach((entity) => {
+			expect(entityGuId(entity)).toBe(entity.id);
+		});
+	});
+});
+
+describe('entityHasDbId', () => {
+	it('should return true if entity has dbId', () => {
+		datetimes.forEach((entity) => {
+			expect(entityHasDbId(entity.dbId)(entity)).toBe(true);
+		});
+	});
+});
```

Comment:
Tried to add a case when it should return false, but TS is not allowing me to do so:
![image](https://user-images.githubusercontent.com/1477453/72607961-4fe46800-392a-11ea-868f-2485c3a14cef.png)

Reply:
In that case, you may try this:
```js
const newEntity: any = omit(['dbId'], entity);
```
or this:
```ts
const newEntity = omit<any, string>(['dbId'], entity);
```

Reply:
First option worked:
![image](https://user-images.githubusercontent.com/1477453/72623097-2b4db780-394d-11ea-8fda-cd033a43e187.png)
***
Thx